### ISSUE TITLE
Actual Cheat News 

### ISSUE THREAD

**Issue by SURBEFAN** [IceHacks/SurvivCheatInjector#567]
Hello there! There are some actual cheat news!

***

Now cheat is not working, because the new update of the anticheat. It blocks entering the menu.

There is a war between SURVIV.IO and hackers. Surviv updates the anticheat and ICEHACKS update the hacks.

Maybe this hack is completely blocked ... But I hope the best)

Anyway! I think both sides work really slow, so make the avantage and implement it.

***

**To summarise:**

Every thing is gonna be ok ... if ICEHACKS will check  [the issues](https://github.com/IceHacks/SurvivCheatInjector/issues)  more often.

Best wishes

**Comment by SURBEFAN** [comment]
Write what you think here!

**Comment by yeeter2003** [comment]
how can it be fixed

**Comment by dinhanhx** [comment]
hm. freezing then green screen then menu not working, there is a pattern here. I think we are reaching the limit. Next will be the website not loading with cheat.

**Comment by BigQuantumMan** [comment]
They just obfuscated the code. I don't know if anyone will want to fix it...

**Comment by yeeter2003** [comment]
it would be sexc if someone can fix it is there an other cheat that works now?

**Comment by SURBEFAN** [comment]
i will write another news with more info if there is gonna be smth interesting) Thx for active

**Comment by SURBEFAN** [comment]
@IceHacks how about writing [ZombsRoyale.io](zombsroyale.io) soft??? LOL
Shold be cool XD

**Comment by 068scar** [comment]
yo stop hacking at surviv better to play like norm player cuz whats the point ?!!

**Comment by dinhanhx** [comment]
I play Server Asia because I'm from Vietnam. In that server, in solo mode, people always team up. Number of teamer in 1 match can up to 8. (Fuck you, Vietnamese players) That's why I have to hack to win.

**Comment by 068scar** [comment]
@dinhanhx  wow what a great reason for hacking good job

**Comment by dinhanhx** [comment]
@068scar is that a sarcasm?

**Comment by FarTooJunior** [comment]
Yea, my entire website doesn't even load with ICEHACKS anymore. I do remember the good glory days of icehacks baby steps though, and to see it go is sad.

**Comment by Peter-Dodge** [comment]
icehacks sometimes works for me, like i can see through everything, and my gun radius shows, and it detects enemies, but the aimbot wont work

**Comment by dinhanhx** [comment]
> icehacks sometimes works for me, like i can see through everything, and my gun radius shows, and it detects enemies, but the aimbot wont work

Can you NOT reply to closed issue? It's dead. Nobody will fix the tool.
